tox-integration.ini

```diff
@@ -42,8 +42,8 @@ commands =
 
     mqtt: tavern-ci --stdout test_mqtt.tavern.yaml
     mqtt: python -c "from tavern.core import run; exit(run('test_mqtt.tavern.yaml', pytest_args=[]))"
-    mqtt_failures: tavern-ci --stdout test_mqtt_failures.tavern.yaml
-    mqtt_failures: python -c "from tavern.core import run; exit(run('test_mqtt_failures.tavern.yaml', pytest_args=[]))"
+    mqtt: tavern-ci --stdout test_mqtt_failures.tavern.yaml
```

Comment:
Will this work with duplicate keys?

Reply:
It's ot a 'key', its just saying which tests to run under which environment